<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-VODB96N" ignored="2" name="Default test" tests="1" failures="1" timestamp="14 ינו 2018 22:18:49 GMT" time="6.261" errors="0">
  <testcase name="testBuyerRegistrtion" time="6.117" classname="com.affilomnia.leadbaskets.BuyerTest">
    <failure type="java.lang.reflect.UndeclaredThrowableException">
      <![CDATA[java.lang.reflect.UndeclaredThrowableExceptionat com.sun.proxy.$Proxy12.download(Unknown Source)at Pages.BuyerPage.integrationPage(BuyerPage.java:227)at com.affilomnia.leadbaskets.BuyerTest.testBuyerRegistrtion(BuyerTest.java:50)at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)at java.lang.reflect.Method.invoke(Unknown Source)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.Invoker.invokeMethod(Invoker.java:571)at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:707)at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:979)at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)at org.testng.TestRunner.privateRun(TestRunner.java:648)at org.testng.TestRunner.run(TestRunner.java:505)at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)Caused by: javax.net.ssl.SSLHandshakeException: Received fatal alert: handshake_failureat sun.security.ssl.Alerts.getSSLException(Unknown Source)at sun.security.ssl.Alerts.getSSLException(Unknown Source)at sun.security.ssl.SSLSocketImpl.recvAlert(Unknown Source)at sun.security.ssl.SSLSocketImpl.readRecord(Unknown Source)at sun.security.ssl.SSLSocketImpl.performInitialHandshake(Unknown Source)at sun.security.ssl.SSLSocketImpl.startHandshake(Unknown Source)at sun.security.ssl.SSLSocketImpl.startHandshake(Unknown Source)at org.apache.http.conn.ssl.SSLConnectionSocketFactory.createLayeredSocket(SSLConnectionSocketFactory.java:396)at org.apache.http.conn.ssl.SSLConnectionSocketFactory.connectSocket(SSLConnectionSocketFactory.java:355)at org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:142)at org.apache.http.impl.conn.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:359)at org.apache.http.impl.execchain.MainClientExec.establishRoute(MainClientExec.java:381)at org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:237)at org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:185)at org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:89)at org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:111)at org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:185)at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:83)at com.codeborne.selenide.impl.DownloadFileWithHttpRequest.executeHttpRequest(DownloadFileWithHttpRequest.java:90)at com.codeborne.selenide.impl.DownloadFileWithHttpRequest.download(DownloadFileWithHttpRequest.java:56)at com.codeborne.selenide.commands.DownloadFile.execute(DownloadFile.java:29)at com.codeborne.selenide.commands.DownloadFile.execute(DownloadFile.java:18)at com.codeborne.selenide.commands.Commands.execute(Commands.java:143)at com.codeborne.selenide.impl.SelenideElementProxy.dispatchAndRetry(SelenideElementProxy.java:87)at com.codeborne.selenide.impl.SelenideElementProxy.invoke(SelenideElementProxy.java:63)... 28 more]]>
    </failure>
  </testcase> <!-- testBuyerRegistrtion -->
  <testcase name="testLoginTotalSpentAndLeads" time="0.0" classname="com.affilomnia.leadbaskets.BuyerTest">
    <ignored/>
  </testcase> <!-- testLoginTotalSpentAndLeads -->
  <testcase name="testTotalLeadTotalSpentAvgcplParCamp" time="0.0" classname="com.affilomnia.leadbaskets.BuyerTest">
    <ignored/>
  </testcase> <!-- testTotalLeadTotalSpentAvgcplParCamp -->
</testsuite> <!-- Default test -->
